packages/backend/test/panels/YeomanUIPanel.spec.ts

```diff
@@ -53,13 +57,17 @@ describe("YeomanUIPanel unit test", () => {
   });
 
   describe("runGenerator", () => {
+    afterEach(() => {
+      getProcessVersionsMock.restore();
+    });
```

Comment:
remove. redundant. you created the free mock instance on `beforeEach`, but use the `.verify()` to verify your [expectations](https://sinonjs.org/releases/latest/mocks) instead of.